server/v2/stf/core_router_service.go

```diff
@@ -87,7 +59,20 @@ func (m *msgRouterService) InvokeUntyped(ctx context.Context, msg protoiface.Mes
 	return m.handler(ctx, msg)
 }
 
-var _ router.Router = (*queryRouterService)(nil)
+// NewQueryRouterService implements router.Service.
+func NewQueryRouterService(queryRouterBuilder *MsgRouterBuilder) router.Service {
+	queryRouter, err := queryRouterBuilder.Build()
+	if err != nil {
+		panic("cannot create queryRouter")
```

Comment:
maybe include the error message in the panic?
```suggestion
		panic(fmt.Errorf("cannot create msgRouter: %w", err))
```